frontend/package.json

```diff
@@ -142,11 +145,18 @@
     }
   },
   "browserslist": {
-    "production": [">0.2%", "not dead", "not op_mini all"],
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
     "development": [
       "last 1 chrome version",
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "remove-markdown": "^0.5.0"
```

Comment:
Due to the way vite bundles dependencies, and for consistency sake, I think this belongs in `devDependencies` instead.

Reply:
yes, this was a lazy mistake by me.